_apps/weekly-digest.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Weekly Digest
+description: Weekly Automated Summary of activities and happening on your GitHub repository
```

Comment:
> Weekly Automated Summary of activities and happening on your GitHub repository

Looks like there is a typo here? 
I'd also stick to sentence case. Maybe something like this?
"Weekly automated summary of activity on your GitHub repository"

Just a suggestion of course